release-notes.txt

```diff
@@ -1,3 +1,30 @@
+OpenMDAO Version 1.7.0 Alpha Release Notes
+June 30, 2016
+
+Backwards-Incompatible Change:
+* Change fd_options to be more self consistent, changed name to deriv_options.
+
+Features:
+* NAS access component
+* Derivatives output adds an asterisk to the 'Absolute Error' and 'Relative Error'
+  lines if those lines show a violation greater than some threshold value (1.0E-6 by default).
+* User can now specify scaler and resid_scaler in add_output and add_state metadata.
+* A host of improvements to the model structure viewer.
+* Updated our Kriging based on Sci-kit learn's Kriging
+* Added support for parallel DOEs using multiprocessing.
+* Converted NREL Tutorial from OpenMDAO 0.1x to OpenMDAO 1.x
+
```

Comment:
(add) The default Line Search for the Newton solver is now None. The Backtracking Line Search has been overhauled to use the Armijo-Goldstein for termination.